### ISSUE TITLE
Rendering ShapeNet Objects with Multiple Textures -- Issue with Loading Atlas

### ISSUE THREAD

**Issue by b7leung** [facebookresearch/pytorch3d#381]
## 🐛 Bugs / Unexpected behaviors

Is it possible to render objects with multiple textures? I wasn't sure if #79 indicated this feature has been implemented yet or not. Otherwise, this is either a bug, or I am doing things wrong. When I try loading a shapenet mesh with textures, I get the following error: 

"ValueError: Expected atlas to be a list of tensors of shape (F, R, R, D) with the same value of R."

This is because aux.texture_atlas returns None for some reason. This issue doesn't show up when I try to open the cow .obj in the examples.

## Instructions To Reproduce the Issue:

I am using the .obj and .mtl from a shapenet chair:

[f9cdefd31b167f95e8f8c11a24c52ebb.zip](https://github.com/facebookresearch/pytorch3d/files/5311728/f9cdefd31b167f95e8f8c11a24c52ebb.zip)

When opened in meshlab, we can see that some of the parts have texture (they're red).
![SNAG-0002](https://user-images.githubusercontent.com/20059131/94806372-b15a4300-03a2-11eb-8939-80455b7ace62.png)

Then, the code I'm using to try and load a mesh for rendering is:
```
from pytorch3d.io import load_obj
from pytorch3d.renderer.mesh import TexturesAtlas
from pytorch3d.structures.meshes import Meshes

mesh_path = "f9cdefd31b167f95e8f8c11a24c52ebb/model.obj"
verts, faces, aux = load_obj(mesh_path, load_textures=True, create_texture_atlas=True)
mesh = Meshes(
    verts=[verts],
    faces=[faces.verts_idx],
    textures=TexturesAtlas(atlas=[aux.texture_atlas]),
)
```

**Comment by nikhilaravi** [comment]
@b7leung currently the texture atlas feature only works with materials which are defined by texture images and/or material properties. When I opened your example data files, the materials in the `.mtl` file are only defined in terms of material properties . In the `mtl` loading code we don't yet support this feature (see https://github.com/facebookresearch/pytorch3d/blob/master/pytorch3d/io/mtl_io.py#L57-L59) 

We could update this function to support your use case - i.e. if there are no texture images, in the texture atlas set the color of each face to the diffuse color of the material. We already do this but it's after the `if` check for the texture images: 
https://github.com/facebookresearch/pytorch3d/blob/master/pytorch3d/io/mtl_io.py#L79-L92

**Comment by b7leung** [comment]
@nikhilaravi thank you for the response. I'm still not very familiar with the implementation details; would this be fixed by a simple change to mtl_io.py (and if so, what would be the change)? Or is this a more complicated issue that would take a longer time to integrate into pytorch3d?

**Comment by nikhilaravi** [comment]
This would be a simple change. I can add a fix for this.

**Comment by nikhilaravi** [comment]
@b7leung this has been fixed in f5383a7e5a79bd9e912af0fb0199c557b9987877.
